telesto/app.py

```diff
@@ -26,21 +26,11 @@ def __init__(self):
 
     @staticmethod
     def _load_model():
-        components = os.environ.get("MODEL_CLASS").split(".")
-
-        if components is None:
-            raise EnvironmentError("MODEL_CLASS environment variable not defined, please specify it in the Dockerfile")
-
-        module_name = ".".join(components[:-1])
+        module_name = "model"
         module = import_module(module_name)
-        class_name = components[-1]
+        class_name = "ClassificationModel"
         model_class = getattr(module, class_name)
-        model_path = os.environ.get("MODEL_PATH")
-
-        if model_path is None:
-            raise EnvironmentError("MODEL_PATH environment variable not defined, please specify it in the Dockerfile")
-
-        return model_class(model_path=model_path)
+        return model_class()
```

Comment:
If we're removing the dynamic class name selection logic, I might make more sense just to use
```
import model

class PredictResource:
    def __init__(self):
        self.model_wrapper = model.ClassificationModel()
```

Reply:
I think we cannot really do this, since the `model` module is not part of the base image. This is added by the expert.

Reply:
If you start this script only inside the final image, it should work. We also wanted to have an example model in this repository, didn't we?

Reply:
We have, it is in the `tests` folder.